### PR TITLE
Add support for random intervals

### LINKED ISSUE #89: Add random time feature?
Would it be possible to add random time so for example you could have 10:H that would do it at a random minute past 10?

### PR DESCRIPTION
**Pull request by gilbsgilbs**
Hi!

This PR supersedes @grampajoe work on #26. I rebased master and fixed `Job` class repr as pointed out by @eth0izzle . Closes #89.

Any chance this gets released anytime soon?

Best,
Gilbsgilbs

### CODE REVIEW

**Comment by gilbsgilbs** [comment]
@dbader Any news on this? I'd need this for one of my projects.

**Comment by grampajoe** [comment]
This is great, thanks for taking the time to finish this up! :+1:

File: schedule/__init__.py [hunk 1]
```diff
@@ -344,6 +356,15 @@ def at(self, time_str):
         self.at_time = datetime.time(hour, minute)
         return self
 
+    def to(self, latest):
+        """Schedule the job to run at an irregular interval.
+
+        The job's interval will vary from the value given to `every` to
+        `latest`.
```
Comment (dbader) [inline schedule/__init__.py]: Might want to add a note here to make it clear that the range is inclusive on both ends due to the use of `randint()`, that is `every(A).to(B).seconds` executes every N seconds such that `A <= N <= B`.

**Comment by dbader** [comment]
Great job, LGTM!

**Comment by dbader** [comment]
@grampajoe and @gilbsgilbs: By the way, I added both of you to AUTHORS.rst. Thanks for the team work 🙂

**Pull request merged**